apps/explorer-e2e/src/integration/transactions.cy.js

```diff
@@ -0,0 +1,87 @@
+//Tests set to skip until transactions are generated after capsule start up
+context.skip('Transactions page', function () {
+  before('visit token home page', function () {
+    cy.visit('/');
+  });
+
+  describe('Verify elements on page', function () {
+    const transactionNavigation = 'a[href="/txs"]';
+    const transactionRow = 'transaction-row';
+    const txHash = 'hash';
+
+    beforeEach('Navigate to transactions page', function () {
+      cy.get(transactionNavigation).click();
+    });
+
+    it('transactions are displayed', function () {
+      cy.get(transactionRow).should('have.length.above', 1);
+    });
+
+    it('transactions details page is displayed', function () {
+      clickTopTransaction();
+      validateTxDetailsAreDisplayed();
+    });
+
+    it('transactions details page is displayed in mobile', function () {
+      cy.common_switch_to_mobile_and_click_toggle();
+      cy.get(transactionNavigation).click();
+      clickTopTransaction();
+      validateTxDetailsAreDisplayed();
+    });
+
+    function clickTopTransaction() {
+      // eslint-disable-next-line cypress/no-unnecessary-waiting
+      cy.wait(1000); // Wait for transactions to load if there are any
```

Comment:
suggestion: cy.get with timeout would be better

Reply:
Originally it was done this way but we didn't want tests failing when there were no transactions. The tests using this function are not running at the moment because there are no transactions in Capsule. This will probably be refactored when we get transactions in